debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/spi/Snapshotter.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.postgresql.spi;
+
+import io.debezium.connector.postgresql.PostgresConnectorConfig;
+import io.debezium.relational.TableId;
+
+import java.util.Optional;
+
+public interface Snapshotter {
```

Comment:
Can you add a class-level comment? It's a good idea for all the types in the SPI package so implementors understand about their semantics.